commonknowledge/meep-intelligence-hub
# Hub uses 2024 constituencies

Switches Hub to query and display 2024 constituency data.

## How Can It Be Tested?

1. If you haven't already ingested 2023/4 WMCs, run `python manage.py import_areas`
2. Go to your local hub's `/map` route and query a postcode
3. Things should appear!

## How Will This Be Deployed?

- [x] Import WMC23 in prod
